architecture/additional_concepts/storage.adoc

```diff
@@ -492,6 +492,183 @@ spec:
+classes will be set as the default. For example, a cluster provisioned in
+us-east-2 would have three storage classes that are named to indicate their
+availability zone.
+
+.StorageClass availability zone
+[cols="3,2,1",options="header"]
+|===
+|StorageClass Name |Provisioner |Age
+|gp2-encrypted-us-east-2a (default)
+|kubernetes.io/aws-ebs
+|3d
+|gp2-encrypted-us-east-2b
+|kubernetes.io/aws-ebs
+|3d
+|gp2-encrypted-us-east-2c
+|kubernetes.io/aws-ebs
+|3d
+|===
+
+[[osd-multi-az-pv-usecases]]
+=== Use cases
+
+[[osd-multi-az-pv-usecases-pvc-storageclass-cli]]
+==== Creating a PersistentVolumeClaim with a specific storage class (CLI)
+
+Creating a PersistentVolumeClaim (PVC) via the CLI requires a manifest file that
+provides the name and details of the PersistentVolume (PV) being requested.
+To have the PVC use a different storage class, simply indicate that in your
+manifest.
+
+.PVC example
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mypvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+  requests:
+    storage: 1Gi
+  storageClassName: gp2-encrypted-us-east-2b <1>
+----
+<1> Specify your storage class to override the default.
+
+Passing this manifest to `oc create -f <manifest filename>` will create a PV in
+the AZ of the storage class provided. You can confirm this by checking the
+output of `oc get pvc`.
+
+[[osd-multi-az-pv-usecases-pvc-storageclass-ui]]
+==== Creating a PersistentVolumeClaim with a specific storage class (UI)
+
+The web console UI makes choosing a storage class for your PVC easy.
+In the application console, navigate to the Storage dashboard and click the
+"Create Storage" button. The first option on the "Create Storage" page is a
+dropdown list of available storage classes to select from.
+
+image::storage-class.png[Create storage with UI,title="Storage Class dropdown"]
+
+The dropdown menu shows the name, zone, and access mode for the available
+storage classes. You can select your desired storage class, provide a name and
+size for the PVC, and click "Create".
+
+[[osd-multi-az-pv-usecases-pvc-non-default-storageclass-cli]]
+==== Using non-default storage classes with OpenShift templates (CLI only)
+
+OpenShift ships with a number of templates that can be used to quickly deploy
+popular applications and frameworks. These are listed in the catalog when using
+the web console and are viewable from CLI by viewing the output of.
+
+----
+$ oc get templates -n openshift
+----
+
+Some of these templates include PVC definitions to provide persistent storage,
+but they do not natively support selection of a non-default storage class
+(however there is an RFE filed to support this in the future).
+To make use of a template with a specific storage class, you'll need to copy the
+template to disk and edit it prior to using it. For example, if you wanted to
+use the MongoDB template with a PVC in us-east-2b, you would first copy the
+template to a file.
```

Comment:
us-east-2b: 
    
    .Copy the template to a file:
    +
    ----
    $ oc get template -n openshift mongodb-persistent -o yaml --export > mongodb-persistent.yml
    ----
   
    .  Look through . . .